src/Client.php

```diff
@@ -310,6 +319,10 @@ public function getInvoiceMarkup($invoiceId, $type)
 
     public function __doRequest($request, $location, $action, $version, $one_way = 0)
     {
+        if (strpos($action, 'getZoneInfo') === false && $this->version >= 1.6 && empty($this->__default_headers[0]->data['IntegrationCode'])) {
```

Comment:
Formatting this line for readability.
```suggestion
        if (strpos($action, 'getZoneInfo') === false &&
            $this->version >= 1.6 &&
            empty($this->__default_headers[0]->data['IntegrationCode'])
        ) {
```